pages/animestatic/[animedetailstatic].js

```diff
@@ -53,9 +44,8 @@ export async function getStaticPaths(){
 export async function getStaticProps(context){
     const {params}=context
     const {animedetailstatic}=params
-    const response=await fetch(`https://ghibliapi.herokuapp.com/films?id=${animedetailstatic}`)
-   // const response=await fetch(`https://ghibliapi.herokuapp.com/films/${params.animedetailstatic}`)
-    const data=await response.json()
+    const url=`https://ghibliapi.herokuapp.com/films?id=${animedetailstatic}`
+   const data=await ApiCall(url)
```

Comment:
cant we call the apiCall and in useeffect and render the data then, and have a loader before the data is fetched? @deepcode3